recipes-extended/ltp/ltp/0004-build-Add-option-to-select-libc-implementation.patch

```diff
@@ -39,18 +39,19 @@ index 297f8e7..906b280 100644
  # COMMON_TARGETS:	Targets which exist in all, clean, and install.
  # INSTALL_TARGETS:	Targets which exist in clean and install (contains
 diff --git a/testcases/kernel/Makefile b/testcases/kernel/Makefile
-index 50a12fa..4f1987f 100644
+index 3319b3163..94ea362c8 100644
 --- a/testcases/kernel/Makefile
 +++ b/testcases/kernel/Makefile
-@@ -49,12 +49,15 @@ SUBDIRS			+= connectors \
+@@ -49,13 +49,16 @@ SUBDIRS			+= connectors \
  			   logging \
  			   mem \
  			   numa \
 -			   pty \
  			   sched \
  			   security \
- 			   timers \
```

Comment:
where did the timers tests go?

Reply:
Was removed by LTP in 0051eab232e7f777f435945b1aba8911f203d65b.